connection.js

```diff
@@ -124,7 +125,9 @@ function connect(id, password, roomName) {
          *
          */
         function connectionFailedHandler(error, message, credentials, details) {
-        /* eslint-enable max-params */
+            _logJwtErrors(message);
```

Comment:
What about mobile?

Reply:
Moved the logging under the `base/conferencemiddleware.any.js`  handler for connection failed.